pkg/util/pki/fetchdata.go

```diff
@@ -0,0 +1,35 @@
+package pki
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
```

Comment:
Using go 1.21 this package was deprecated and fails the golangci-lint test. Changing to `io` only, works.
```io/ioutil" has been deprecated since Go 1.19: As of Go 1.16, the same functionality is now provided by package io or package os, and those implementations should be preferred in new code. See the specific function documentation for details.  (staticcheck)```

Reply:
Thank you. Done.